### ISSUE TITLE
New feature request: Navigate new Explore page for web version.

### ISSUE THREAD

**Issue by digitizesuccess** [shine-jayakumar/insta-likecom-bot#92]
When using the hashtag utility, IG now only posts the top 12-13 tags on the web version of the Explore Page. However, they have added a new Explore Page where the hashtag isn't used. For instance, #Denver displays only 12 posts out of over 14,000,000, but Denver works like the old Explore page where you can keep moving through them. Can we get a new function that allows the bot to navigate through that page as opposed to the hashtag page? Or in conjunction with it?

**Comment by imakou** [comment]
Yeah, this is a critical change IG recently made. And it would fail the flow..

**Comment by ap0l0** [comment]
This is the path that allows infinite posts:

`https://www.instagram.com/explore/search/keyword/?q=<keyword here or tag>
`

**Comment by ap0l0** [comment]
@shine-jayakumar was that URL implemented? I can see it in the code but I don't know which parameter to use. I am trying to like posts of a tag or location.

**Comment by ap0l0** [comment]
It works for me and it has infinite scroll:

https://www.instagram.com/explore/search/keyword/?q=rock%20climber

<img width="1328" alt="image" src="https://github.com/shine-jayakumar/insta-likecom-bot/assets/14894967/dd58331e-22b8-4541-bcb4-cdb1cf6d1b01">

**Comment by shine-jayakumar** [comment]
@ap0l0 Interesting. It doesn't seem to work for me. I tried a different account too. Some region restriction?

![Screenshot from 2023-12-01 08-20-10](https://github.com/shine-jayakumar/insta-likecom-bot/assets/84611104/534de376-96cc-45af-9956-3d8abd89f083)

**Comment by ap0l0** [comment]
Interesting, did you try with another keyword? or try going to https://www.instagram.com/explore/search/ to see what loeads, and which keywords at the top

That function has infinite scroll

**Comment by ap0l0** [comment]
@shine-jayakumar does it work when you get keywords from your account? I tried with two other accounts and it seems to work

**Comment by digitizesuccess** [comment]
> @ap0l0 Interesting. It doesn't seem to work for me. I tried a different account too. Some region restriction?
> 
> ![Screenshot from 2023-12-01 08-20-10](https://private-user-images.githubusercontent.com/84611104/287132778-534de376-96cc-45af-9956-3d8abd89f083.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0OTM3NjgsIm5iZiI6MTcwMTQ5MzQ2OCwicGF0aCI6Ii84NDYxMTEwNC8yODcxMzI3NzgtNTM0ZGUzNzYtOTZjYy00NWFmLTk5NTYtM2Q4YWJkODlmMDgzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAyVDA1MDQyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTg2Mzg1YTkwZDlmMzliY2Q5YjA2NWQ3MmUwMzZkMDJkZGVlN2Q5MjNlMmNmYWQzZDFjZTJhMzc5ZDUyZTJjODYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.qRwZUvQOLSxpxxs0Yvly0mLDPxiYC5049jyJTM57CGs)

same here, when I run a script it only finds accounts it doesn't find the new search explore page

**Comment by shine-jayakumar** [comment]
Opened a discussion regarding this: [Navigate new Explore page for web version #98](https://github.com/shine-jayakumar/insta-likecom-bot/discussions/98)
